config/routes.rb

```diff
@@ -6,5 +6,9 @@
   resources :people
   resources :home, only: [:index]
 
-  root to: "home#index"
+  root :to => "dashboards#show"
+  get "/login" => redirect("/auth/github"), as: :login
+
+  get '/auth/github/callback', to: 'sessions#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.